joint_limits_interface/include/joint_limits_interface/joint_limits_interface.h

```diff
@@ -111,6 +111,27 @@ class PositionJointSaturationHandle
     }
 
     const double cmd = internal::saturate(jh_.getCommand(), min_pos, max_pos);
+
+    // Optional helper code for debugging, not realtime safe ----------
+#define USE_JOINT_LIMIT_DEBUG // Show warnings when joint limits exceeded
```

Comment:
I have the impression that this `#define` should be defined in the CMakeLists.txt as a cmake option (https://cmake.org/cmake/help/v3.0/command/option.html):

``` cmake
option(BUILD_WITH_JOINT_LIMIT_DEBUG "Build with Joint Limit Debugging mode" OFF)
if(BUILD_WITH_JOINT_LIMIT_DEBUG)
  add_definitions(-DUSE_JOINT_LIMIT_DEBUG)
endif(BUILD_WITH_JOINT_LIMIT_DEBUG)
```